library/traefik

```diff
@@ -51,5 +51,5 @@ Constraints: nanoserver-ltsc2022, windowsservercore-ltsc2022
 Tags: v2.11.3, 2.11.3, v2.11, 2.11, mimolette
 Architectures: amd64, arm32v6, arm64v8, ppc64le, s390x
 GitRepo: https://github.com/traefik/traefik-library-image.git
-GitCommit: 98edda3a6008ef1b658911e757b68bd06ea2ed8d
+GitCommit: e971e752e97b340b47dde038edbb9052d3064781
```

Comment:
If not waiting for https://github.com/traefik/traefik-library-image/pull/62, this needs to be reverted